### PR TITLE
Change content by clicking on it

### LINKED ISSUE #33: Change content by clicking on it
Give the user the possibility to change by clicking :
- the name of the board
- the list name 
- the card name (in the large modal)
- the card content (in the large modal)

### PR DESCRIPTION
**Pull request by jrmneveu**
fix #33

### CODE REVIEW

File: desktop/app/scss/pulpe-structure/pulpe-board.scss [hunk 1]
```diff
@@ -14,6 +14,30 @@
...
+		input.board-title {
+			display   : inline-block;
+			width     : calc(100% - 250px);
+			background: white;
+			border    : none;
+			padding   : 0 0 0 5px;
+			margin    : 3px 0 0;
+			font-style: italic;
+
+			&.is-not-displayed {
```
Comment (asdine) [inline desktop/app/scss/pulpe-structure/pulpe-board.scss]: `hidden` may be a better name for that

**Comment by jrmneveu** [comment]
@asdine waiting for the "list title" feature to finish this one.

**Comment by jrmneveu** [comment]
@asdine 👍

**Comment by asdine** [comment]
@jrmneveu The PR is ready to be reviewed. The scss of the card modal has been slightly changed, check on the `Files changed` view.

**Comment by jrmneveu** [comment]
@asdine 👍

**Pull request merged**